org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServerWrapper.java

```diff
@@ -322,10 +322,13 @@ public synchronized void start() throws IOException {
 				FileBuffers.getTextFileBufferManager().addFileBufferListener(fileBufferListener);
 			}).exceptionally(e -> {
 				LanguageServerPlugin.logError(e);
-				initializeFuture.completeExceptionally(e);
 				stop();
-				return null;
+				throw new RuntimeException(e);
```

Comment:
is there a reason for now just `throw e`?

Reply:
Yes, unfortunately e is of type `Throwable` and `Function.apply` can only throw an unchecked exception.